packages/playground/src/dashboard/components/create_farm.vue

```diff
@@ -92,11 +94,17 @@ export default {
         isCreating.value = false;
       }
     }
+    function validateFarmName(name: string) {
+      if (!name.split("").every((c: string) => /[a-zA-Z0-9\-_]/.test(c))) {
+        return { message: "Invalid Farm Name" };
```

Comment:
I suggest to make the message contain the name rules,  what characters are allowed and so on

Reply:
done, thanks